src/adr/0039-distributed-blob-garbage-collector.md

```diff
@@ -1,4 +1,4 @@
-# 29. Distributed blob garbage collector
+# 39. Distributed blob garbage collector
```

Comment:
Why do we jump from 29 -> 39?

Reply:
Because:
 - there was no number available before
 - I prefer keeping eventBus related ADR close together.
 - It's still "in progress"